README.md

```diff
@@ -1,7 +1,8 @@
 # MorseProject
 
-This Project is going to be created in the context of the lecture 'Software Engineering' in the winter semester 2021 in the course Applied Computer Science ('Angewandte Informatik') of the DHBW Mannheim.
-The goal is the transmission of Morse code between two or more devices via sound.
+This Project is going to be created in the context of the lecture 'Software Engineering' in the winter semester 2021 in
+the course Applied Computer Science ('Angewandte Informatik') of the DHBW Mannheim. The goal is the transmission of
+Morse code between two or more devices via sound.
```

Comment:
Same text only with breaks, but looks okay. We need to add the contributors later anyway.